swedish/start.md

```diff
@@ -3,5 +3,10 @@ Ett förfärligt oväsen - mörkret lättar.
 Du har sovit dåligt, förföljd i dina mardrömmar hela natten. Du stänger väckarklockan och sätter
 dig upp. Drömmen, vad hände egentligen? Allting känns så främmande.
 
-Främmande, precis som dina egna tankar. Du inser att du tänker på engelska och blir förvirrad.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.